src/website/app/demos/Icon/index.js

```diff
@@ -27,6 +27,7 @@ export default {
   doc,
   examples,
   slug: 'icon',
+  syntax: `import <IconName> from 'mineral-ui-icons/<IconName>'`,
```

Comment:
You failed to document how to import the generic Icon component.

```jsx
import Icon from `mineral-ui/Icon`;
```

Reply:
I was wondering about that. It seemed weird to show it in the header, and using `Icon` directly is a special case that's already called out in the Custom example on that page.